numpy/lib/histograms.py

```diff
@@ -345,6 +345,97 @@ def _search_sorted_inclusive(a, v):
         a.searchsorted(v[-1:], 'right')
     ))
 
+def histogram_bin_edges(a, bins=10, range=None, weights=None):
+    """
+    Function to calculate only the edges of the bins used by the histogram function. 
```

Comment:
If you put backticks around the word `histogram`, it will create a link to the function in the docs

Reply:
Done.